DevEdu.API/DevEdu.Business/ValidationHelpers/GroupValidationHelper.cs

```diff
@@ -18,5 +20,21 @@ public void CheckGroupExistence(int groupId)
             //if (group == default)
             //    throw new EntityNotFoundException(string.Format(ServiceMessages.EntityNotFoundMessage, nameof(group), groupId));
         }
+        public void CheckUserBelongToGroup(int groupId, int userId, Role role)
+        {
+            var userGroupId = _groupRepository.GetUser_GroupByUserIdAndUserRoleAndGroupId(userId, role, groupId);

```

Comment:
please don't create such stored procedures
just imagine how many SPs will we get with this approach

you can ask in chat which methods might be used